### PR TITLE
Set watchtower to update/watch only shadowbox container

### PR DESCRIPTION
**Pull request by tanelpuhu**
(or others with `com.centurylinklabs.watchtower.enable=true` label). Might fix issue #48. 

Same as in #48 happened to me - after installation many running containers got *upgraded* automatically but thankfully nothing broke. After this change, installed server again and seems to be working just fine, non of the other images have been pulled automatically and containers *upgraded*.

### CODE REVIEW

File: src/server_manager/install_scripts/install_server.sh [hunk 1]
```diff
@@ -260,6 +260,7 @@ function start_shadowbox() {
   # rather than pass in the environment.
   declare -a docker_shadowbox_flags=(
     --name shadowbox --restart=always --net=host
+    --label=com.centurylinklabs.watchtower.enable=true
```
Comment (JonathanDCohen) [inline src/server_manager/install_scripts/install_server.sh]: A disadvantage of this approach is that existing shadowbox containers won't get the label unless there's a manual re-install.  

Is it possible we could add the label directly to the Dockerfile?  Then when we next  push shadowbox all the existing containers will get this label automatically.

Comment (tanelpuhu) [reply src/server_manager/install_scripts/install_server.sh]: Indeed...  if stable image is created from `src/shadowbox/docker/Dockerfile`, perhaps something similar might help:
```
diff --git src/shadowbox/docker/Dockerfile src/shadowbox/docker/Dockerfile
index f5c1ec6..9601f42 100644
--- src/shadowbox/docker/Dockerfile
+++ src/shadowbox/docker/Dockerfile
@@ -24,2 +24,5 @@ LABEL shadowbox.outline-ss-server_version="${SS_VERSION}"

+# Set watchtower to automatically fetch docker image updates
+LABEL com.centurylinklabs.watchtower.enable=true
+
 ARG GITHUB_RELEASE
```
And label here would also be used for custom `SB_IMAGE` but not sure if that is desired behaviour 🤔 maybe it'll be local in most cases...

Comment (tanelpuhu) [reply src/server_manager/install_scripts/install_server.sh]: Moved the label to Dockerfile

**Comment by JonathanDCohen** [comment]
LGTM, I'll let you hit the merge button.  Thanks for the contribution!

File: src/shadowbox/docker/Dockerfile [hunk 2]
```diff
@@ -25,6 +25,9 @@ LABEL shadowbox.outline-ss-server_version="${SS_VERSION}"
 ARG GITHUB_RELEASE
 LABEL shadowbox.github.release="${GITHUB_RELEASE}"
 
+# Set watchtower to automatically fetch docker image updates
+LABEL com.centurylinklabs.watchtower.enable=true
```
Comment (JonathanDCohen) [inline src/shadowbox/docker/Dockerfile]: On discussion with the team offline, we decided that this is too much of a layering violation. Sorry, could you please move this back to the install script?

Comment (tanelpuhu) [reply src/shadowbox/docker/Dockerfile]: No worries, moved back to install_server.sh

**Pull request merged**